Source/WebKit/WebProcess/WebPage/haiku/WebPageHaiku.cpp

```diff
@@ -121,8 +121,9 @@ const char* WebPage::interpretKeyEvent(const KeyboardEvent* event)
     return 0;
 }
 
-String WebPage::platformUserAgent(const URL&) const
+String WebPage::platformUserAgent(const URL& url) const
 {
+	notImplemented();
```

Comment:
This sounds easy to implement.

You can try "Mozilla/5.0 (Haiku; Intel R1beta1) AppleWebKit/536.30.1 (KHTML, like Gecko) Version/6.0.5 WebPositive/2.0"